src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -711,7 +727,9 @@ class ConsentGroupReview extends Component {
     consentGroup.institutionalSources = JSON.stringify(this.getInstitutionalSrc(this.state.formData.instSources));
     consentGroup.describeConsentGroup = this.state.formData.consentExtraProps.describeConsentGroup;
     consentGroup.requireMta = this.state.formData.consentExtraProps.requireMta;
-
+    if(this.state.reviewSuggestion) {
```

Comment:
nit: spacing here and elsewhere: 
```suggestion
    if (this.state.reviewSuggestion) {
```